giles-eco/src/main/webapp/WEB-INF/views/uploads/showDocument.jsp

```diff
@@ -129,4 +152,17 @@
 </div>
 </c:if>
 </div>
-</div>
\ No newline at end of file
+</div>
+<script>
+$('#reProcessDocument').on('show.bs.modal', function (event) {
+	  var button = $(event.relatedTarget); // Button that triggered the modal
+	  var docId = button.data('docid'); // Extract info from data-* attributes
+	  // If necessary, you could initiate an AJAX request here (and then do the updating in a callback).
+	  // Update the modal's content. We'll use jQuery here, but you could use a data binding library or other methods instead.
+	  var modal = $(this)
+	  var url = '<c:url value="/documents/" />';
+	  url += docId;
+	  url += "/reprocess";
+	  modal.find("#submitReprocessDocument").attr('action', url);
```

Comment:
I don't think you need javascript to fill the url attribute. It's the page for the document, right? So just one document id, you can just add the url to the form above.